### PR TITLE
Add 12 icons

### PR DESCRIPTION
**Pull request by manasch**
## Description

Add 12 icons, listed below

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Icons added
* Comic Geeks (`com.LeagueofComicGeeks.app2`)
* cult.fit (`fit.cure.android`)
* Dimmer (`giraffine.dimmer`)
* DiskUsage (`com.google.android.diskusage`) - added from icon request form
* English (`livio.pack.lang.en_US`)
* FastHub-RE (`com.fastaccess.github.revival`)
* Flashcards (`flashcards.words.words`)
* Literal (`com.literal`)
* Nekome (`com.chesire.nekome`)
* StoryGraph (`com.thestorygraph.thestorygraph`)
* Trakt (`tv.trakt.trakt`)
* Vyom (`com.infrasoft.uboi`)

### CODE REVIEW

**Comment by manasch** [comment]
I've been working on some other icons, if this hasn't been accepted by the time I'm done I'll add it to this if that's fine.

**Comment by x9136** [comment]
Reviewers will definitely tell you it's worth to read [the Lawnicon guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#icon-guidelines) and fix what's wrong.

**Comment by manasch** [comment]
> Reviewers will definitely tell you it's worth to read [the Lawnicon guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#icon-guidelines) and fix what's wrong. 

Hi, I'm sorry that the contributions upset you, I'll try to find what's wrong and fix them.

But, I am going to assume that you are not happy with the icons that have curves as they do not match the 6px round radius.
If not for that, I'll gladly fix anything else that might be pointed out.

Thanks

**Comment by x9136** [comment]
@manasch Hi, I'm not upset. I thought I'd draw your attention to the guidelines ahead of time because the moderators will ask you to do it anyway.

**Comment by manasch** [comment]
@x9136 Hi, I understand sorry.
I am making the changes rn. From what I found, I made the mistake of not using base as 12px and rounded corners not having the 6px radius.

I've switched all to 12px base with the minimal ones getting 14px as well as finer details getting 6px. My only doubt would lie in the usage of the standard widths mentioned.

I will be making another commit with the fixes soon.

I hope this would resolve it.

**Comment by manasch** [comment]
I have made the changes to fit the guidelines. The only one that I see to be rejected is fasthub-re (in which I will give up on that, I do not know how I can draw that better :smiling_face_with_tear:).
I used 8px as the default stroke as 12px was too thick. As for the finer details I used 6px.

I have ensured that the rest of them use the base stroke as 12px as much as possible along with 14px for the ones that are minimal.

I hope this fits to the standard.
Thanks.

**Comment by x9136** [comment]
@manasch Cool, but you forgot to round corners somehow. 🥲 There are no right angle icons in this pack.

**Comment by manasch** [comment]
That is actually quite embarrassing. Sorry for the repeated work, I have changed them all to have rounded corners.
I actually failed to understand from the guidelines what rounded corners meant.

Sorry again.

**Review by Chefski — CHANGES_REQUESTED** [review]
Just quickly looking over the icons I see they all have a white background. Could you change it to transparent. I'll review them more later

**Comment by manasch** [comment]
@Chefski Hey, before I make another commit, was just checking for any other changes to be made and noticed that the svg's I made have newlines and the color is mentioned as `black` whereas the ones already existing in the repo have no newlines (everything in oneline) and color as `#000`.

Would this be an issue and if so do I manually change it?  I've been using figma to create and export them and I don't see any export setting to get it the same way as the existing icons.

File: svgs/english_dictionary.svg
Comment (Chefski) [inline svgs/english_dictionary.svg:1]: Change to fit 160x160 content area. Make sure the fill shape still has the acceptable line width.
image

File: svgs/fasthub_re.svg
Comment (Chefski) [inline svgs/fasthub_re.svg:1]: Fit to 160x160 content area
image

File: svgs/flashcards.svg
Comment (Chefski) [inline svgs/flashcards.svg:1]: Fit to 160x160 content area

File: svgs/literal.svg
Comment (Chefski) [inline svgs/literal.svg:1]: Fit to 160x160 content area

File: svgs/nekome.svg
Comment (Chefski) [inline svgs/nekome.svg:1]: Fit to 160x160 content area

File: svgs/story_graph.svg
Comment (Chefski) [inline svgs/story_graph.svg:1]: Fit to 160x160 content area

File: svgs/vyom.svg
Comment (Chefski) [inline svgs/vyom.svg:1]: Fit to 160x160 content area

**Review by Chefski — CHANGES_REQUESTED** [review]
Some icons don't quite fit the content area. I fixed 3. If you could just fit them to the 160x160 content area then they will be fine :)

File: svgs/english_dictionary.svg
Comment (manasch) [reply svgs/english_dictionary.svg:1]: The fill shape is 6px, but not sure if I can accurately depict the flag, I'll commit the changes I've made but if it's not acceptable I'll remove it.

**Review by Chefski — APPROVED** [review]

**Pull request merged**